R/sleuth.R

```diff
@@ -180,11 +180,10 @@ sleuth_prep <- function(
   msg('')
 
   check_result <- check_kal_pack(kal_list)
-  if ( length(check_result$no_bootstrap) > 0 ) {
-    stop("You must generate bootstraps on all of your samples. Here are the ones that don't contain any:\n",
-      paste('\t', kal_dirs[check_result$no_bootstrap], collapse = '\n'))
-  }
-
+  #if ( length(check_result$no_bootstrap) > 0 ) {
```

Comment:
this needs to be fixed and should be implemented in `read_bootstrap_statistics()`